### DISCUSSION TITLE
Danish Translation

### DISCUSSION THREAD

**Discussion by sorentorp** [ChristopherKlay/StadiaEnhanced#81] [Translations]
LANG: da
 
"default": "Standard",
"native": "Hjemmehørende",
"hide": "Skjul",
"show": "Vs",
"visible": "Synlig",
"hidden": "Skjult",
"enabled": "Aktiveret",
"disabled": "Deaktiveret",
"auto": "Automatisk",
"manual": "Manuelt",
"responsive": "Lydhør",
"windowed": "Vindue-tilstand",
"fullscreen": "Fuld skærm",
"searchstore": "Søg i butik",
"onsale": "På Udsalg",
"prodeals": "Pro Tilbud",
"allgames": "Alle spil",
"usermedia": "Skærmbilleder og videoer",
"searchbtnbase": "Søg videre",
"avatarpopup": "Ny avatar-URL (tom for standard):",
"searchheader": "Spil inkluderet",
"sessiontime": "Sessionstid",
"codec": "Codec",
"resolution": "Opløsning",
"trafficsession": "Sessionstrafik",
"trafficcurrent": "Nuværende trafik",
"trafficaverage": "Gennemsnitlig trafik",
"packetloss": "Tabte pakke",
"framedrop": "Rammer tabt",
"latency": "Netværksventetid",
"jitter": "Jitter Buffer",
"compression": "Kompression",
"streammon": "Overvågning af strøm",
"stream": "Strøm",
"community": "Fællesskab",
"speedtest": "Hastighedstest",
"quickaccess": "Hurtig adgang",
"messages": "Beskeder",
"avatar": "Inkarnation",
"interface": "Brugerflade",
"shortcut": "Genveje",
"shortcuttitle": "Installer en genvej for",
"shortcutdesc": "Giver dig mulighed for at installere en genvej til et spil på din enhed",
"gridsize": "Gitterstørrelse",
"griddesc": "Ændrer antallet af spil pr. række på startskærmen.",
"clock": "Ur",
"clockdesc": "Viser det aktuelle tidspunkt på vennelisten",
"friendslist": "Venneliste",
"igoverlay": "Overlay i spillet",
"listoverlay": "Liste & Overlay",
"filter": "Filter",
"filterdesc": "Giver dig mulighed for at sortere din startskærm ved at skjule spil. Filteret kan skiftes via symbolet",
"filtertoggle": "Skift",
"filterquick": "Hurtig",
"invitebase": "Kopier invitationslink",
"inviteactive": "Kopieret!",
"prolabel": "Pro-mærke",
"prolabeldesc": "Fjerner 'Pro' -mærket fra spil på startskærmen.",
"homegallery": "Brugergalleri",
"homegallerydesc": "Skjuler området 'Optager' nederst på startskærmen.",
"quickprev": "Eksempel på besked",
"quickprevdesc": "Skjuler beskedeksemplet på vennelisten.",
"quickrep": "Hurtigt svar",
"quickrepdesc": "Skjuler hurtigsvaret i chats.",
"offlinefriend": "Offline venner",
"offlinefrienddesc": "Skjuler offline venner på vennelisten.",
"invisiblefriend": "Usynlige venner",
"invisiblefrienddesc": "Skjuler venner med ukendt onlinestatus på vennelisten.",
"streammode": "Streaming Tilstand",
"streammodedesc": "Aktiver for at gøre visse elementer (dvs. vennelisten) ulæselige under streaming (via værktøjer som OBS / Discord).",
"catprev": "Eksempel på kategori",
"catprevdesc": "Skjuler kategoritags, når du svæver over et spil.",
"popup": "Popup-effekt",
"popupdesc": "Fjerner zoom-in / forstørrelseseffekten, når du svæver over et spil på hjemmemarkedet.",
"streammondesc": "Aktiver for at starte skærmen, når et spil starter.",
"resolutiondesc": "Den målrettede opløsning til spil. 1440p og 2160p kræver VP9.",
"codecdesc": "Den codec, der bruges til spil.",
"confirmreset": "Er du sikker på, at du vil nulstille indstillingerne?",
"gamesfinished": "Gennemførte spil",
"achievementsunlocked": "Oplåste Præstationer",
"splitstore": "Opdel butikslister",
"splitstoredesc": "Opdeler butikslister i to kolonner for at få et bedre overblik.",
"scrollbar": "Rullebjælker",
"scrollbardesc": "Aktiverer rullebjælker på startskærmen og gem.",
"resetsettings": "Nulstil indstillingerne"

**Comment by ChristopherKlay** [comment]
Thank you!

Will be included in the next update (:

**Comment by ChristopherKlay** [comment]
Small update of the translation file for `1.8.4` [here](https://github.com/ChristopherKlay/StadiaEnhanced/blob/master/translations.md#new-content).
No stress however; i'll update translation when available (:

**Reply by sorentorp** [reply]
Okay. Let me know if you need any additions...

**Reply by ChristopherKlay** [reply]
There has been a couple of [new values](https://github.com/ChristopherKlay/StadiaEnhanced/blob/master/translations.md#new-content), everything else works fine! (:

**Reply by sorentorp** [reply]
"games": "Spil"
"bundles": "Bundter"
"addons": "Tilføjelser"
"hardware": "Hardware"
"software": "Software"
"decodetime": "Afkodningstid"

**Reply by ChristopherKlay** [reply]
Thank you! (:
